run.py

```diff
@@ -20,14 +22,24 @@
 
 @main_router.message(Command(commands=["start", "help"]))
 async def start(message: types.message):
+    user_telegram_id = message.chat.id
+    user_telegram_username = message.from_user.username
+
     all_categories_button = types.KeyboardButton(text=Localizator.get_text_from_key("all_categories"))
     my_profile_button = types.KeyboardButton(text=Localizator.get_text_from_key("my_profile"))
     faq_button = types.KeyboardButton(text=Localizator.get_text_from_key("faq"))
     help_button = types.KeyboardButton(text=Localizator.get_text_from_key("help"))
-    keyboard = [[all_categories_button, my_profile_button], [faq_button, help_button]]
+    checkout_button = types.KeyboardButton(text=Localizator.get_text_from_key("checkout"))
+
+    # TODO: insert item count here, these code lines are not correct yet
+    #product_count = await CartService.get_all_cart_items_count(user_telegram_id)
+    #cart_button_label = Localizator.get_text_from_key("cart") + " (" + str(product_count) + ")"
```

Comment:
The final code must not contain commented lines of code

Reply:
I'll leave the TODO in if you agree